### DISCUSSION TITLE
Filament plugins

### DISCUSSION THREAD

**Discussion by CodeTechNL** [njxqlus/filament-relation-manager-component#5] [Q&A] [answered]
Thank you for this package, it works perfectly! Why isn't it at the plugins list of the official Filament website?

**Comment by njxqlus** [comment, marked as answer]
@CodeTechNL thank you!

It's not well tested and it's "out of concept of Filament" I think. Maybe I'll add it later

**Comment by momostafa** [comment]
Another big Thank you for this brilliant package it works like charm out of the box...!
really should be featured on official Filament website.
would be awesome if you can add a feature to create the relation file pre filed with from and table schema instead of manually copying everything over and over   
Filament v3.2.86
PHP 8.2.19
Laravel 11.10.0

**Comment by Wraxton** [comment]
Another big thanks here @njxqlus.

Great package, have not experienced any issues using it. 

Would love to see similar functionality as a plugin or even implemented in core.
